### PR TITLE
Add function to lint APK files.

### PR DESCRIPTION
**Pull request by Elizafox**
No functional changes (yet). This only adds a function to lint APK files.

### CODE REVIEW

File: pkg/linter/linter.go [hunk 1]
```diff
@@ -344,3 +349,41 @@ func (lctx LinterContext) LintPackageFs(fsys fs.FS, warn func(error), linters []
...
+	}
+
+	lctx := NewLinterContext(pkgname, apkfs)
+
+	err = lctx.LintPackageFs(apkfs, warn, linters)
+	if err != nil {
+		return err
+	}
+
+	return nil
```
Comment (imjasonh) [inline pkg/linter/linter.go:388]:
```suggestion
	return lctx.LintPackageFs(apkfs, warn, linters)
```

**Review by imjasonh — COMMENTED** [review]
One small cleanup, otherwise lgtm!

**Comment by Elizafox** [comment]
Thanks @imjasonh :).

File: pkg/linter/linter.go (hunk 1 above)
Comment (Elizafox) [reply pkg/linter/linter.go]: This is done. 👍

**Review by imjasonh — APPROVED** [review]

**Pull request merged**